### ISSUE TITLE
WHEN HANDING IN CHEMICAL PT4 TO SKIER, PRAPOR'S MAKE-UP QUEST DOES NOT TRIGGER

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#907]
version 3.5.7 
I saw someone had this same issue but never received an answer. I completed the quest "out of curiosity" and unlocked "loyalty buyout" but not "No offense". The only thing i can think is i never accepted "big customer" and that might have caused it. 

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/44) by _**spentz**_

**Comment by refringe** [comment]
`No Offence` should appear after failing `Big customer`, which is what you did, right?

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/44#issuecomment-18522) by _**chomp**_

**Comment by refringe** [comment]
TL;DR: Works as intended; you're meant to accept Out of Curiousity and Big Customer before you can disappoint Therapist and Praper for their respective "apology" quests (Loyalty Buyout & No Offense)

So the thing is, if you don't *accept* Big Customer and just let it go, Papa Praps' follow-up quest (No Offense) doesn't start, and because Big Customer isn't actually started then it doesn't actively pop as failed on finishing either Thrapist or Skier's variant of Chem P4. At least from my experiences for, like, years of EFT, it's been this way. It happens for other quests with optional alternative paths, especially just casually modded without a workaround for this issue in mind.

If someone can confirm the opposite on Live, then that'd be lovely, but BSG haven't fixed this "bug" since the quests were first implemented. Big Customer and Out of Curiousity just get silently changed to the failed state, preventing their follow-up quests (No Offense and Loyalty Buyout, respectively) from being unlocked and also failed, pretty much nullifying the resulting reputation losses and needing to make it up to Praps and Therapist.

Wanna know a fun fact? Out of Curiousity unlocks after completing Part 3 and you can completely ignore Chemical - Part 4 for no bad effect with Skier from failing Chem P4, as a result Big Customer and No Offense is utterly bypassed with no penalties, Chem P4 is silently failed with no penalties and life carries on.

Testing;
So made a fresh Edge of Darkness USEC account on 3.6.0, setup the quest chains for Chemical - Part 3 to be active and finishable in-game. Subtasks for Out of Curiousity and Big Customer pinged, but neither started properly. Accepted Part 4, Big Customer is now an acceptable quest.

Did Part 4 without accepting Big Customer OR Out of Curiousity: Both Big Customer and Out of Curiousity re-locked, as expected; neither No Offence nor Loyalty Buyout appears in the quest lists.
Complete Part 4 w/ Big Customer accepted: No Offence is startable, as normal.
Complete Part 4 w/ Out of Curiousity accepted: Big Customer is silently failed and No Offence doesn't pop up, as normal.

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/44#issuecomment-18639) by _**Sister-Fister**_

**Comment by refringe** [comment]
Closing as a year old and most quests have changed in different ways, if this is still an issue, start a new issue

---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/44#issuecomment-26760) by _**CWX**_
